docs/guides/LiveCode Builder Language Reference.md

```diff
@@ -575,6 +575,64 @@ includes the specification of function signature, according to the
 [standard rules for forming these](http://journals.ecs.soton.ac.uk/java/tutorial/native1.1/implementing/method.html) 
 when calling the JNI.
 
+The function `interface` may be used on Android to create an interface 
+proxy - that is an instance of a generic Proxy class for a given 
+interface. This effectively allows LCB handlers to be registered as the 
+targets for java interface callbacks, such as event listeners.
+
+The foreign handler binding to such a function takes a value that should 
+either be a `Handler` or an `Array` - if it is a `Handler`, the handler 
```

Comment:
Hmm, ok - do you think we should check using reflection on the Java side that there *is* indeed only on callback in this case?